src/migrations/20190826063213-create-todo.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Todos', {
```

Comment:
Instead of creating a todo table, please create a user table with the same fields we have in the user schema found in models/user.js file. Also I expect that when you are done with this task, you will have sequelize replace mongoose totally and remove the mongoose code as we won't have need for mongoose in the codebase.